.github/workflows/pr-realm-js.yml

```diff
@@ -131,23 +138,9 @@ jobs:
       - name: Setup node version
         uses: actions/setup-node@v4
         with:
-          node-version: 20.11.1
```

Comment:
This was for a workaround in `prebuild` which is not released and this PR includes an upgrade to get the fix.